src/credentials/FabricTable.cpp

```diff
@@ -281,7 +277,8 @@ CHIP_ERROR FabricInfo::SetCert(MutableByteSpan & dstCert, const ByteSpan & srcCe
 }
 
 CHIP_ERROR FabricInfo::VerifyCredentials(const ByteSpan & noc, const ByteSpan & icac, ValidationContext & context,
-                                         PeerId & nocPeerId, FabricId & fabricId, Crypto::P256PublicKey & nocPubkey) const
+                                         FabricId & fabricId, NodeId & nodeId, CompressedFabricId & compressedFabricId,
```

Comment:
Compressed fabric ID should NOT exist here.

Reply:
Removed this argument